source/_integrations/hue.markdown

```diff
@@ -80,12 +80,14 @@ script:
         data:
           group_name: "Porch"
           scene_name: "Porch Orange"
+          transition: 30
 ```
 
 | Service data attribute | Optional | Description                                                           |
 | ---------------------- | -------- | --------------------------------------------------------------------- |
 | `group_name`           | no       | The group/room name of the lights. Find this in the official Hue app. |
 | `scene_name`           | no       | The name of the scene. Find this in the official Hue app.             |
+| 'transition'           | yes      | The time in 100s of milliseconds to transition to the scene.          |
```

Comment:
what does 100s of milliseconds mean?

Reply:
That is how the Hue api is by default, i.e. a value of 4 is 400 milliseconds.

Reply:
Ah! Now I get it. Let's add an example to clarify it for other readers:

```suggestion
| 'transition'           | yes      | The time in 100s of milliseconds to transition to the scene. For example, a value of 4 means 400 milliseconds.          |
```